core/components/atoms/selectionCard/hooks/useMultiSelect.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+export function useMultiSelect() {
+  const [selectedItemList, setSelectedItemList] = React.useState<string[]>([]);
```

Comment:
Please use type Object or Object and String as sometimes the values passed may be complex and it will help in integration.